client/components/team/TeamList.jsx

```diff
@@ -3,29 +3,43 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Card } from 'semantic-ui-react';
 import { times } from 'lodash';
+import MessageUserModal from '../message-user/message-user';
 
 TeamList = class TeamList extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+    constructor(props) {
+	super(props);
+	this.state = { messageUser: null };
+    }
+    
+    render() {
+	const { messageUser } = this.state;
+	return (
+	    <div>
+		<Card.Group>
+		    { this._mapTeams() }
+		</Card.Group>
+		<MessageUserModal user={messageUser} clearUser={() => this._clearMessageUser()}/>
+	    </div>
+	);
+    }
+    
+    _mapTeams() {
```

Comment:
The formatting in this codebase is annoyingly inconsistent. When possible, if you're able to adopt the existing indentation for a file or avoid changing formatting on otherwise-unedited lines, that would be nice (but not necessary). I always like to keep the git blame clean. Not worth changing this file for this PR.

Reply:
I'm only seeing four-space indents in this file. Are you seeing something different?

Reply:
Changed several files to use two-space indents, which seems to have been the general convention before

Reply:
Thanks - not sure what I was seeing before